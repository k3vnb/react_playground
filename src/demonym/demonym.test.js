import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Demonym from './Demonym'

describe('<Demonym /> Component', () => {
    it('renders complete form', () => {
        expect(toJson(shallow(<Demonym />))).toMatchSnapshot()
    })
})