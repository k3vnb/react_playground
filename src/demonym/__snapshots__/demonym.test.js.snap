// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Demonym /> Component renders complete form 1`] = `
<div
  className="demonym"
>
  A 
   comes from 
</div>
`;
